config/swagger.yaml

```diff
@@ -920,18 +920,30 @@ paths:
       security: []
 
 definitions:
+  EncodedHash:
+    type: string
+  Pow:
+    type: array
+    items:
+      type: integer
+      format: int32
```

Comment:
[Unsigned integer 32 bits](https://github.com/aeternity/protocol/blob/f578d480f8c8e13c1425f26ebb732db01d099ab3/CONSENSUS.md#solution-encoding-rules) so `int32` is unable to contain real values sometimes. Options are (in my order of preference):
* format uint32 (not documented in swagger doc, but shall be ok to use it as a hint - see [here](https://github.com/OAI/OpenAPI-Specification/blob/ba49b592c13594d06db4ea99fe4cd795b3d83ace/versions/2.0.md#data-types))
* no format
* format int64 (specified by swagger spec), potentially with min value 0

Reply:
But we only use 4 byte integers... Having int64 blows up the random tests that take any arbitrary integer in that range. It's the min and max in that case we would need to set.

Reply:
I believe that if we have in the past and still used int64 everywhere, then we can run as well with int32 in a format field. And yes, all int64 mentioned before also were unsigned, but this was and is unspecified. This is an unused over specification of what the API can deal with, completely in line how swagger works.